Metagenomics/slides/index.html

```diff
@@ -0,0 +1,144 @@
+<!doctype html>
+<html lang="en">
+
```

Comment:
@shiltemann any chance we can get this in markdown before mergeing? We can do this also post-merge no problem.